packages/frontend/components/authorize/AuthorizeClaim.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+
+
+
+interface Props {
+  claim: keyof typeof claims,
+}
+
+const props = withDefaults(defineProps<Props>(), {})
+
+const claims = {
+    "identity": {
+        "header": "Identity",
+        "description": "View your basic identity info",
+    },
+    "email": {
+        "header": "Email",
+        "description": "View your email address",
+    },
+    "phone": {
+        "header": "Phone",
+        "description": "View your phone number",
+    },
+    "address": {
+        "header": "Address",
+        "description": "View your address",
+    }
+}
+
+// def header
+const header = computed(() => {
+  try{
+    return claims[props.claim].header
+  }
+  catch{
+    throw new Error("Invalid claim")
+  }
```

Comment:
You shouldn't throw error inside a component, there is no way to handle it. We should try to fall back to some logical default, such as not showing the component. Throwing an error could cause problems within our app.

We can use optional chaining, to handle non existent cases.
```suggestion
    return claims[props.claim]?.header
```